src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -144,9 +139,10 @@ namespace ts.codefix {
             }
 
             const declSourceFile = classOrInterface.getSourceFile();
-            const inJs = isSourceFileJS(declSourceFile);
+            const modifierFlags = (makeStatic ? ModifierFlags.Static : 0) | (startsWithUnderscore(token.text) ? ModifierFlags.Private : 0);

```

Comment:
`isPrivateIdentifier`?

Reply:
@amcasey Thanks for the feedback. No, we need to add an action for a name which starts from `_` and make it `private`, 

```ts
class Foo {
  foo() {
     this._bar();
       // Need to show 
         // - Declare private method _bar
         // - Declare method bar
         // ...
  }
}
```

Reply:
> It seems like it was already possible to add a method, public property, or private property - just not a private property

Before 

1. add a method
2. public property, private property if a name starts from `_`

After these changes

1. add a method, private method  if a name starts from `_`
2. public property, private property if a name starts from `_`

What I did in this PR

1. Combined logic for using it for properties and methods
2. Added modifiers flags to avoid creating a new variable for each modifier, like `makeStatic` etc.
3. Simplified methods signatures, instead of passing all properties as separate parameters we can use one parameter with `Info` type
https://github.com/microsoft/TypeScript/blob/deb5bac520a1134e188fc6f01fa4f52b66d2899b/src/services/codefixes/fixAddMissingMember.ts#L212
4. Added `createModifiersFromModifierFlags` to create modifiers instead of create `Tokens` based on the variables
https://github.com/microsoft/TypeScript/blob/deb5bac520a1134e188fc6f01fa4f52b66d2899b/src/services/codefixes/helpers.ts#L247
5. Moved the following logic to two separate methods - `getActionsForMissingMethodDeclaration` and `getActionsForMissingMemberDeclaration`
https://github.com/microsoft/TypeScript/blob/deb5bac520a1134e188fc6f01fa4f52b66d2899b/src/services/codefixes/fixAddMissingMember.ts#L23-L28

Reply:
Thanks!  I find that sort of logical breakdown very helpful.